src/schema/nmdc.yaml

```diff
@@ -2256,7 +2264,45 @@ enums:
       - ORCID:0000-0002-8683-0050 #Montana Smith
     permissible_values:
       V-bottom conical tube:
-
+  
+  FailureWhatEnum:
+    description: The permitted values for describing where a failure occured during processing in the lab during analysis workflows.
+    permissible_values:
+      Low read count:
+        description: Number of output reads is not sufficent to continue to the next analysis step.
+      Malformed data:
+        description: Workflow failure reading input or writing the output file(s).
+      Assembly size too small:
+        description: The size of the metagenome or metatranscriptome assembly is too small to proceed to the next analysis workflow.
+      No valid data generated:
+        description: A process ran but did not produce any output. Ie binning ran but did not produce any medium or high quality bins.
+      Other:
+        description: A lab process or analysis workflow has failed in a way that has not been captured by the avaliable values yet. Please use slot 'comment' to specify details.
+
+  FailureWhereEnum:
+    description: The permitted values for describing where in the process, either a lab or analysis workflow step, the failure occurred. 
+    permissible_values:
+      Omics_Processing:
```

Comment:
It looks like these are shadowing classes - we don't have an easy way to auto-shadow but I'd recommend using the exact same names for consistency, e.g. OmicsProcessing.

Reply:
@cmungall  I was following the style guide for permissible values, I will update to the same names if that is your suggestion. I did have a question about if a Class and permissible value can share the same name but based on yoru comment this is allowed.

Reply:
Permissible values have been updated to match the class names.